### ISSUE TITLE
Erroneously aligned columns on pdbqt -type conversions <-- merge request

### ISSUE THREAD

**Issue by flatstik** [openbabel/openbabel#2595]
Open Babel 3.1.1 (latest github pull)

`obabel -i moo $1.out -o pdbqt -O $1.pdbqt` gives erroneusly aligned columns:

output is following:
```
REMARK                            x       y       z     vdW  Elec       q    Type
REMARK                         _______ _______ _______ _____ _____    ______ ____
ROOT
ATOM      1  C   UNL     1       2.931  -0.773   1.137  0.00  0.00    +3.000 C
ATOM      2  C   UNL     1       1.914  -0.437   2.256  0.00  0.00    +62.000 C
ATOM      3  C   UNL     1       1.881  -1.432   3.414  0.00  0.00    +67.000 C
ATOM      4  C   UNL     1       1.630  -2.850   2.903  0.00  0.00    +172.000 C
...
```

the correct output should be following:

```
REMARK                            x       y       z     vdW  Elec       q    Type
REMARK                         _______ _______ _______ _____ _____    ______ ____
ROOT
ATOM      1  C   UNL     1       2.931  -0.773   1.137  0.00  0.00    +3.000 C
ATOM      2  C   UNL     1       1.914  -0.437   2.256  0.00  0.00   +62.000 C
ATOM      3  C   UNL     1       1.881  -1.432   3.414  0.00  0.00   +67.000 C
ATOM      4  C   UNL     1       1.630  -2.850   2.903  0.00  0.00  +172.000 C
...
```

The partial charges - `q`  - are aligned by `+ or -` which is not ok and also but the type (in this example, carbons) are not aligned as they should.. It happens to every type of atoms.

**Comment by ghutchis** [comment]
Is there actually a spec for PDBQT? Yes, it looks like the charges should be right-aligned, but if you know of a spec, it would be very helpful.

**Comment by flatstik** [comment]
https://web.archive.org/web/20190301030908/https://autodock.scripps.edu/faqs-help/faq/what-is-the-format-of-a-pdbqt-file

https://userguide.mdanalysis.org/1.1.1/formats/reference/pdbqt.html

**Comment by flatstik** [comment]
I recon the line 406 of pdbqtformat.cpp should be 
```    snprintf(buffer, BUFF_SIZE, "%s%5d %-4s %-3s %c%4d%c   %8.3f%8.3f%8.3f  0.00  0.00    %6.3f %-2.2s", ```

**Comment by ghutchis** [comment]
Fixed by #2601 - now merged, thanks!
